build-system/runner/src/org/ampproject/AmpPass.java

```diff
@@ -48,6 +48,8 @@
   private final Map<String, Node> prodAssignmentReplacements;
   final boolean isProd;
 
+  final Integer ampExtensionCallbackPosition = 4;
```

Comment:
Add javadoc here to describe what the signature looks like.

Also, why not just go with the "last" arg?